ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressInput.component.ts

```diff
@@ -79,7 +79,11 @@ import { LoggingService } from '../../services/logging.service';
         </mat-form-field>
 
         <ng-container *ngIf="ais.countryInfo$ | async as info; else defaultStateField">
-          <ng-container *ngIf="hasStatesList(info); then showStateDropdown else showStateTextField">
+          <ng-container *ngIf="!info.stateCode">
+            <ng-container *ngIf="info.subnationalDivisionTypeName as stateLabel; else defaultStateField">
```

Comment:
#minor we probably don't need `as stateLabel`

Reply:
I would keep it as is.
Because it helps us understand that we use `subnationalDivisionTypeName` field from data model as to display `stateLabel` in the form.